fedimint-server/src/config/api.rs

```diff
@@ -180,8 +178,13 @@ impl ConfigGenApi {
         })
     }
 
-    /// Once configs are generated, starts DKG and await its
-    /// completion, calling a second time will return an error
+    /// Once configs are generated, updates status to ReadyForConfigGen and
+    /// spawns a task to coordinate DKG, then returns. Coordinating DKG in a
+    /// separate thread allows clients to poll the server status instead of
+    /// blocking until completion, which can be fragile due to timeouts, poor
+    /// network connections, etc.
+    ///
+    /// Calling a second time will return an error.
```

Comment:
checking for `SharingConfigGenParams` or `ReadyForConfigGen` status would allow success returns instead of error when api is called multiple times